samples/system-test/productSearch.v1p3beta1.test.js

```diff
@@ -16,20 +16,22 @@
 'use strict';
 
 const path = require(`path`);
+const uuid = require(`uuid`);
 const vision = require('@google-cloud/vision').v1p3beta1;
 const productSearchClient = new vision.ProductSearchClient();
 const assert = require('assert');
 const tools = require(`@google-cloud/nodejs-repo-tools`);
 const cmd = `node productSearch.v1p3beta1.js`;
-const cwd = path.join(__dirname, `..`);
+const cwd = path.join(__dirname, `..`, `productSearch`);
 
 // Shared fixture data for product tests
 const testProductSet = {
   projectId: process.env.GCLOUD_PROJECT,
   location: 'us-west1',
   productCategory: 'homegoods',
-  productId: 'test_product_id_1',
-  productSetId: 'test_product_set_id_1',
+  productId: `test_product_id${uuid.v4()}`,
```

Comment:
Do we have cleanup code that goes around and deletes any created resources?  Want to make sure we're not leaking resources into the test projects.